src/app/inventory/store/d2-store-factory.service.ts

```diff
@@ -126,7 +126,7 @@ const StoreProto = {
     const allItems = (this.items as D2Item[])
       .filter((item) => item.canBeInLoadout())
       // tslint:disable-next-line:no-unnecessary-callback-wrapper
```

Comment:
should we consider turning this rule off in config so we don't need to use the override?

Reply:
I generally like it but it doesn’t work in this one case because it gets oasses a second argument that isn’t what it expects.